### ISSUE TITLE
Devices.json cannot be downloaded from wirelessthings.net any more

### ISSUE THREAD

**Issue by erdmenchen** [WirelessThings/WirelessThings-LaunchPad#55]
Since Wireless Things is shutting down, the configuration wizard cannot be used because it wants to download a file from their domain to update the devices.json file.
It fails to download and the software cannot be started.

Fix:
Update the config of the configuration wizard to download the file from this repository instead:
https://raw.githubusercontent.com/WirelessThings/WirelessThings-LaunchPad/master/ConfigurationWizard/Devices.json

**Comment by Mythandar** [comment]
I made a fork of this and made that change to mine.  Good find.

**Comment by Mythandar** [comment]
https://raw.githubusercontent.com/WirelessThings/devices/master/Devices.json

May be a more recent version that the one in the Launchpad.

**Comment by erdmenchen** [comment]
Yes, you are right. This seems to be a more up to date version of the devices.json.
